### PR TITLE
[mmloader]Update mmloader to v1.0.1

### PR DESCRIPTION
**Pull request by tishion**
## Fix:
1. Fixed C/CXX flags in camake files 
2. Ignore case when comparing string variable (to support vs 2012)

## Tripletes:
Support Windows & x86 & x64 & static (no change)

**This PR follows the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)**

### CODE REVIEW

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: versions/m-/mmloader.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "030c251ff729f1063950f5473cf393125f888ca2",
+      "version-date": "2021-12-13",
+      "port-version": 0
+    },
+    {
+      "git-tree": "1401132ad7398509b08fc506e0ba5b3196b77a8c",
+      "version-string": "2021-12-13",
+      "port-version": 1
+    },
```
Comment (JonLiu1993) [inline versions/m-/mmloader.json:12]:
```suggestion
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: versions/m-/mmloader.json (hunk 1 above)
Comment (tishion) [reply versions/m-/mmloader.json:12]: Addressed already

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
Feature "shellcode" tested successfully in the x64-windows-static

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**